ai/repository/chroma_repository.py

```
@@ -0,0 +1,189 @@
+        embeddings = self.embeddings.embed_documents(texts)
+        
+        # ChromaDB에 추가
+        collection.add(
+            embeddings=embeddings,
+            documents=texts,
+            metadatas=metadatas,
+            ids=ids
+        )
+
+    def exists_collection(self, book_id: str) -> bool:
+            collection_name = self._get_collection_name(book_id)
```

Flake8 (7.2.0)

[error] over-indented

(E117)